helm/kubernetes-elastic-stack-elastic-logging/requirements.yaml

```diff
@@ -1,25 +1,30 @@
 dependencies:
 - name: elasticsearch
-  version: "0.1.8"
+  version: "0.2.1"
   repository: https://giantswarm.github.com/sample-catalog
   condition: elasticsearch.enabled
   
 - name: fluentd-elasticsearch
-  version: "0.1.4"
+  version: "0.2.2"
   repository: https://giantswarm.github.com/sample-catalog
   condition: fluentd-elasticsearch.enabled
 
 - name: kibana
-  version: "0.1.4"
+  version: "0.2.0"
   repository: https://giantswarm.github.com/sample-catalog
   condition: kibana.enabled
 
-- name: keycloak-gatekeeper
-  version: "1.1.1-3"
-  repository: https://giantswarm.github.com/sample-catalog
-  condition: keycloak-gatekeeper.enabled
+# - name: keycloak-gatekeeper
+#   version: "1.1.1-3"
+#   repository: https://giantswarm.github.com/sample-catalog
+#   condition: keycloak-gatekeeper.enabled
 
 - name: elasticsearch-exporter
   version: "0.1.0"
   repository: https://giantswarm.github.com/sample-catalog
-  condition: elasticsearch-exporter.enabled
\ No newline at end of file
+  condition: elasticsearch-exporter.enabled
+
+- name: elasticsearch-curator
+  version: "2.0.3"
+  repository: https://giantswarm.github.com/sample-catalog
+  condition: elasticsearch-curator.enabled
```

Comment:
Super nitpicky and existing bug but please add a newline here.

Reply:
Done